### ISSUE TITLE
Code in Dialogseekbar.xml

### ISSUE THREAD

**Issue by sergiocos** [pix/aeon#112]
for tv episode thumb the code looks like this:
    <texture diffuse="thumbs/cdcover_mask.png" background="true">$INFO[VideoPlayer.Cover]</texture>
    <aspectratio scalediffuse="false">scale</aspectratio>
    <bordertexture border="-32,-7,-32,-7">thumbs/poster_shadow.png</bordertexture>

My guess is it should use masks with correct AR

Example:

```
thumbs/multiplex_tvmask.png (for texture diffuse)
thumbs/episode_poster_shadow.png (for bordertexture).
```

**Comment by sergiocos** [comment]
line 424 and 426, because GitHub doesn't like code :)

**Comment by pix** [comment]
Sergiocos: Github syntax: http://daringfireball.net/projects/markdown/syntax :)

**Comment by pix** [comment]
Closed by a66ca58459e867f6e8ecf5363054ac2143b3c02c: Use se masks with correct aspect ratio
